source/includes/export-connection-warning.rst

```diff
@@ -0,0 +1,3 @@
+When you export favorite connections, database usernames and
+passwords are included in plaintext unless you :ref:`encrypt the
```

Comment:
This renders properly, but 'encrypt the' has a strange underline in GitHub. Maybe just a GH artifact?